docs/Home/history.md

```diff
@@ -0,0 +1,15 @@
+# History
```

Comment:
I think we should replace this History page with something more like "Motivation".

The narrative history isn't too interesting here, and may come off a little pretentious describing before/after as if they are industry shifts that have taken place already. What seems interesting is the motivation for creating a new format:
* must support arbitrary message serializations (like ros2, unlike ros1)
* must support efficient access over a slow network (like ros1, unlike ros2)
* must support heterogeneous types (like ros*, unlike avro, parquet [without abuse/giant union])
* must be self-contained

I think this section would be stronger if it were framed as "here's the requirements, here's the solutions that exist, here's mcap & how it solves", rather than "history of the format".